### PR TITLE
period selection added for fetchPlayer

### PR DESCRIPTION
**Pull request by Hunam6**
Usage exemple: `funcraft.fetchPlayer('_Hunam','month')`

> _month_ can be replaced by _lastMonth_, _last_, _1_ or _true_.

### CODE REVIEW

**Comment by Hunam6** [comment]
@Androz2091 can you review this please?

File: structures/Player.js [hunk 1]
```diff
@@ -11,58 +11,29 @@ const LandRushStats = require("./LandRushStats");
...
-                playerName.children[1].children[2].data.trim();
-            } catch (e) {
-                if (e instanceof TypeError) {
-                    this.grade = '';
-                    for (let i = 1; i < playerName.children.length - 2; i++) this.grade += playerName.children[i].children[0].data;
-                    error = true;
-                }
-            }
-            if (!error) this.grade = playerName.children[1].children[2].data.trim();
-        }
```
Comment (Androz2091) [inline structures/Player.js]: Why did you remove player grade?

Comment (Hunam6) [reply structures/Player.js]: What? I'll fix this

Comment (Hunam6) [reply structures/Player.js]: Fixed 👍

**Review by Androz2091 — APPROVED** [review]
👍
